api/api/projects_api_test.go

```diff
@@ -369,14 +376,64 @@ func TestListProjects(t *testing.T) {
 
 func TestUpdateProject(t *testing.T) {
 	testCases := []struct {
-		desc             string
-		projectID        models.ID
-		existingProject  *models.Project
-		expectedResponse *Response
-		body             interface{}
+		desc                  string
+		projectID             models.ID
+		existingProject       *models.Project
+		expectedResponse      *Response
+		expectedMessage       map[string]interface{}
```

Comment:
Why is there a need to create a new expected output `expectedMessage`? Can't we keep the same `expectedResponse`?